src/Unity.Mathematics/Geometry/MinMaxAABB.cs

```diff
@@ -198,18 +198,6 @@ public void Encapsulate(float3 point)
             Max = math.max(Max, point);
         }
 
-        [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static implicit operator MinMaxAABB(AABB aabb)
```

Comment:
If this is released to public you need to add:

`[Obsolete("<what happend> (RemovedAfter <today + 3 months in yyyy-mm-dd format>).")]`

Reply:
All of this stuff is internal so I don't think I need to mark this as obsolete yet, but there are versions of this code in Entities which I plan to delete, so I probably need to mark as obsolete over there?